### PR TITLE
create learning profile banner 

### PR DESCRIPTION
**Pull request by youssefezzahi96**
[ticket](https://go1web.atlassian.net/jira/software/projects/CLXP/boards/626?selectedIssue=CLXP-2)
purpose: 
is to create learning profile banner component.
things done: 
component & test
to be discussed/checked: 
uploading images on S3 as all other banners, or change the ways things are done (local images)
Visual Results (story book):
image

### CODE REVIEW

**Review by wernerdurand — APPROVED** [review]
Nickel 👌

File: packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  };
+
+  return (
+    <div className={style.root} data-name="start-learning-profile-banner">
+      <div className={style.container}>
+        <div className={style.banner}>
+          <div className={style.left} role="img" aria-label="Left image container">
+            <img
+              className={style.image}
+              src="https://static.coorpacademy.com/site/notebook.png"
```
Comment (silouone) [inline packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js:21]: quand tu m'en parlais je pensais c'etait cote FE que ca allait etre feeder . donc en effet jrejoins adrien, pas ouf aha. 

si tu pouvais au moins le déclarer ds des constantes

File: packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ButtonLink from '../../../atom/button-link';
+import style from './style.css';
+
+const LearningProfileBanner = React.memo(function LearningProfileBanner(props) {
+  const {title, subtitle, cta} = props;
+
+  const BOOK_IMG_URL = 'https://static.coorpacademy.com/assets/images/section-item-book.png';
+  const TARGET_IMG_URL = 'https://static.coorpacademy.com/assets/images/section-item-dart.png';
```
Comment (AdrieanKhisbe) [inline packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js:10]: you can extract the two const out of the LearningProfileBanner

File: packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    ...cta,
+    type: 'primary'
+  };
+
+  return (
+    <div className={style.root} data-name="start-learning-profile-banner">
+      <div className={style.container}>
+        <div className={style.banner}>
+          <div className={style.left} role="img" aria-label="Left image container">
+            <img className={style.image} src={BOOK_IMG_URL} alt="Note book image" />
```
Comment (AdrieanKhisbe) [inline packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js:22]:
```suggestion
            <img className={style.image} src={BOOK_IMG_URL} alt="Notebook image" />
```

File: packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/style.css [hunk 4]
```diff
@@ -0,0 +1,200 @@
...
+  width: 100%;
+  max-width: 1080px;
+  height: 214px;
+  display: flex;
+  justify-content: center;
+  position: relative;
+  top: 0px;
+  left: 0px;
+  background: battle_bg;
+  background-image: url(https://static.coorpacademy.com/site/startBattleBackground.png);
```
Comment (AdrieanKhisbe) [inline packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/style.css:29]: use the generic path I created

**Review by AdrieanKhisbe — CHANGES_REQUESTED** [review]
Few feedbacks

File: packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
+import React from 'react';
```
[the remark this answers is not in the document]
Comment (AdrieanKhisbe) [reply packages/@coorpacademy-components/src/molecule/dashboard/learning-profile-banner/index.js:2]: 👍

**Review by AdrieanKhisbe — APPROVED** [review]

**Review by silouone — APPROVED** [review]

**Pull request merged**